lib/git_hooks/installer.rb

```diff
@@ -1,33 +1,42 @@
 module GitHooks
   class Installer
-    attr_accessor :hook
-
     HOOK_SAMPLE_FILE = 'hook.sample'
 
-    def initialize(hook)
+    def initialize(hook, ruby_path = nil)
       @hook = hook
+      @ruby_path = ruby_path
     end
 
     def install(force = false)
-      return true if installed?
+      throw Exceptions::UnknownHookPresent.new(hook) if !force && installed?
+
+      hook_script = hook_template
```

Comment:
Man, this is not good.

what do you think about create one other class HookTemplate?
All the replacement should be in this class.

IMO Installer class should not be responsable for generating the HookTemplate.